darts/utils/timeseries_generation.py

```diff
@@ -308,6 +308,65 @@ def random_walk_timeseries(mean: float = 0.,
     return TimeSeries.from_times_and_values(index, values, freq=freq, columns=pd.Index([column_name]))
 
 
+def autoregressive_timeseries(coef: Sequence[float],
```

Comment:
How about `coefs` ?

Reply:
Thought about naming it `coefs` but then found that `coefs` is way less popular than `coef` as a parameter name. Also sklearn uses `coef_` to get the model coefficients. But yes coefs would make it clearer that we can take multiple coefficients...

Reply:
OK, we can leave `coef` then.